### PR TITLE
christina-roperto/c04-iac02

### PR DESCRIPTION
**Pull request by christina-roperto**
# Description

Exercise (e.g. C04-E02): 

Student Name: Christina Roperto

> creating VPC and stuff through terraform. This is a fun one, it's nice to see what we did on exercise 02 can be done faster through iac. 

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

**Comment by christina-roperto** [comment]
Ok. Readme file is now added :)

**Comment by christina-roperto** [comment]
Hi Denis, let me know if the problem is fixed :)

File: classes/04class/exercises/c04-iac02/christina-roperto/output.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+output "subnets" {
```
Comment (caiocezart) [inline classes/04class/exercises/c04-iac02/christina-roperto/output.tf:1]: I didn't know that you could do something like that!

File: classes/04class/exercises/c04-iac02/christina-roperto/vpc.tf [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_subnet" "public" {
+  count = 2
+
+  vpc_id            = aws_vpc.devops-iac02.id
+  cidr_block        = cidrsubnet(aws_vpc.devops-iac02.cidr_block, 8, count.index * 2 + 100) # even networks for public subnets
```
Comment (caiocezart) [inline classes/04class/exercises/c04-iac02/christina-roperto/vpc.tf:18]: another one that I had no idea that exists!

**Pull request merged**